File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = -1;
        String ar = Integer.toString(num);
        String[] arr = ar.split("");
        
        for(int i=0;i<arr.length;i++){
            if(arr[i].equals(Integer.toString(k))) return i+1;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int num, int k) {
    int answer = 0;
    int cnt = 1;
    
    while(num > 0){
        if(num % 10 != k){
            cnt++;       
        }else{
            answer = cnt;
        }
        num /= 10;
    }
    
    if(answer == 0) return -1;
    else return cnt - answer +1;
}